content/blog/configure-isut-and-ams-for-vlcm-based-firmware-updates-on-gen-12-servers.md

```diff
@@ -0,0 +1,107 @@
+---
+title: How to configure iSUT and AMS for vLCM-Based firmware updates on HPE
+  Gen12 servers (High Security Mode)
+date: 2025-10-01T02:30:17.312Z
+author: Vijayakannan M
+authorimage: /img/Avatar3.svg
+disable: false
+tags:
+  - OV4VC
+  - VLCM
+  - COM4VC
+  - hpe-oneview
+  - HPE Gen12
+---
+<style> li { font-size: 27px; line-height: 33px; max-width: none; } </style>
+
+
+
+In this blog post, I’ll show you how to configure iSUT and AMS to enable vLCM-based firmware updates on HPE Gen12 servers.
+
+
+
+HPE Gen12 servers introduce enhanced security by supporting only High Security modes (SecureStandard, CNSA, FIPS). This impacts how you configure Intelligent System Update Tool (**iSUT**) and Agentless Management Service (**AMS**) for vSphere Lifecycle Manager **(vLCM)** based firmware updates. 
+
+Unlike previous generations, configuration through the HPE OneView for VMware vCenter (**OV4VC**) and HPE Compute Ops Management plug-in for VMware vCenter (**COM4VC**) vLCM Pre-Check page is not available in these modes, as iLO credentials are now required. Instead, you must manually configure AMS and iSUT by creating an application account and providing valid HPE iLO credentials.
+
+- - -
+
+## **Prerequisites**
+
+* HPE Gen12 server with iLO 7.
+* vSphere environment with vLCM enabled.
+* iLO credentials with sufficient privileges.
+* Access to server CLI (SSH or local console).
+
+- - -
+
+## **Step 1: Create an Application Account on iLO 7**
+
+Application accounts are service accounts in iLO 7, used by host applications (like iSUT and AMS) to securely authenticate and communicate with iLO.
+
+**To create an application account using CLI:**
+
+```shell
+sut appaccount create -u <ilo_username> -p <ilo_password>
+```
+
+**Alternatively, to proceed without creating an application account, provide the iLO credentials using the following CLI command:**
+
+```shell
+sut -set ilousername=<ilo_username> ilopassword=<ilo_password>
+```
+
+- - -
+
+## **Step 2: Set iSUT Mode to AutoDeploy**
```

Comment:
Since AutoDeploy shows up on the screen in what appears to be the name of something, it may be appropriate to use title case here.  My recommended rewrite for this header is:

Step 2: Set iSUT mode to AutoDeploy